docs/content/account-linking/guides/linking-accounts.md

```diff
@@ -2,55 +2,53 @@
 
-In this doc, we’ll dive into a progressive onboarding flow, including the Cadence scripts & transactions that go into its implementation in your dapp. These components will enable any implementing dapp to create a custodial account, intermediate the user’s on-chain actions on their behalf, and later delegate control of that dapp-created account to the user’s wallet-mediated account. We’ll refer to this custodial pattern as the Hybrid Account Model and the process of delegating control of the dapp account as Account Linking.
+In this doc, we’ll dive into a progressive onboarding flow, including the Cadence scripts & transactions that go into its implementation in your dApp. These components will enable any implementing dApp to create a custodial account, intermediate the user’s on-chain actions on their behalf, and later delegate control of that dApp-created account to the user’s wallet-mediated account. We’ll refer to this custodial pattern as the Hybrid Account Model and the process of delegating control of the dApp account as Account Linking.
 
 # Objectives
 
 - Establish a [walletless onboarding flow](https://flow.com/post/flow-blockchain-mainstream-adoption-easy-onboarding-wallets)
 - Create a blockchain-native onboarding flow
 - Link an existing app account as a child to a newly authenticated parent account
-- Get your dapp to recognize “parent” accounts along with any associated “child” accounts
+- Get your dApp to recognize “parent” accounts along with any associated “child” accounts
 - View Fungible and NonFungible Token metadata relating to assets across all of a user’s associated accounts - their wallet-mediated “parent” account and any hybrid custody model “child” accounts
 - Facilitate transactions acting on assets in child accounts
 
 # Terminology
 
-**Parent-Child accounts** - For the moment, we’ll call the account created by the dapp the “child” account and the account receiving its AuthAccount Capability the “parent” account. Existing methods of account access & delegation (i.e. keys) still imply ownership over the account, but insofar as linked accounts are concerned, the account to which both the user and the dapp share access via AuthAccount Capability will be considered the “child” account. This naming is a topic of community discussion and may be subject to change
+**Parent-Child accounts** - For the moment, we’ll call the account created by the dApp the “child” account and the account receiving its AuthAccount Capability the “parent” account. Existing methods of account access & delegation (i.e. keys) still imply ownership over the account, but insofar as linked accounts are concerned, the account to which both the user and the dApp share access via AuthAccount Capability will be considered the “child” account. This naming is a topic of community discussion and may be subject to change
```

Comment:
Given the questions we have heard on the interpretation of parent-child accounts, do you think it would make sense to highlight more details on _how to think about them_ and _what they aren't_ for those?

Reply:
Agreed. Might be helpful to include some mental modeling around what they are and aren't. To prioritize getting docs up, I'll merge what's been approved then follow up with guidance around what you suggested.